File: PathFindingDotnetCore/PathFindingDotnetCore/Models/Cell.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PathFindingDotnetCore.Models
{
    public class Cell : Node
    {
        public int Row { get; }
        public int Col { get; }

        public Cell(int row, int col, int id = -1, bool isStart = false, bool isDestination = false, bool isWall = false)
            : base(id, isStart, isDestination, isWall)
        {
            Col = col;
            Row = row;
        }

        public Node ConvertToNode()
        {
            return new Node(Id, IsStart, IsDestination, IsWall);
        }
    }
}

File: PathFindingDotnetCore/PathFindingDotnetCore/ClientApp/src/components/PathFindingVisualiser/Cell/Cell.js
import React, { Component } from "react";
import { Col } from 'reactstrap';
import "./Cell.css";

export default class Cell extends Component {
    constructor(props) {
        super(props);
        this.state = {};
    }

    render() {
        const {
            height,
            width,
            col,
            row,
            id,
            isDestination,
            isStart,
            isWall,
            onMouseDown,
            onMouseEnter,
            onMouseUp
        } = this.props;
        const extraClassName = isWall ? "node-wall" : isDestination ? "node-destination" : isStart ? "node-start" : "";

        return (
            <div style={{ width: width, height: height }}
                id={`node-${id}`}
                className={`node ${extraClassName}`}
                onMouseDown={() => onMouseDown(row, col)}
                onMouseEnter={() => onMouseEnter(row, col)}
                onMouseUp={() => onMouseUp()}
            ></div>
        );
    }
}
